### ISSUE TITLE
[develop_2.0] Unfolding APIs where Tags have spaces on them

### ISSUE THREAD

**Issue by GerardSoleCa** [swagger-api/swagger-ui#570]
Hi!

I'm testing/using swagger-ui for 2.0 spec. I was trying the examples hosted here: https://github.com/wordnik/swagger-spec/tree/master/examples/v2.0/json

On the example https://github.com/wordnik/swagger-spec/blob/master/examples/v2.0/json/petstore.json#L23 in the remarked line you can see that the element inside tags array has a white space. This brokes the unfolding (opening) api in the UI because its used as id in the markup.

TL;DR: Space on tags brokes the javascript call to fold/unfold apis -> http://stackoverflow.com/questions/6802765/jquery-dealing-with-a-space-in-the-id-attribute

I imagine that a similar issue will happen in <1.2 specs...

Gerard
